cypress/integration/trakt/comments.spec.js

```diff
@@ -0,0 +1,170 @@
+describe('04: Updates, reply and likes on comments.', () => { 
+    before(function(){
+        cy.getAccessToken();
+      });
+    it('04_001: Should post a comment on movie ',()=> {
+        const movie = {
+            "title": "Guardians of the Galaxy",
+            "year": 2014,
+            "ids": {
+              "trakt": 28,
+              "slug": "guardians-of-the-galaxy-2014",
+              "imdb": "tt2015381",
+              "tmdb": 118340
+            }
+        }
+        cy.request({
+            method: 'POST',
+            url: 'https://api.trakt.tv/comments',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${Cypress.env('access_token')}`,
+                'trakt-api-version': Cypress.env('trakt_api_version'),
+              'trakt-api-key': Cypress.env('client_id'),
+            },
+            body: {
+                   movie: movie,
+                   comment: 'I am not the danger!',
+                   spoiler: false
+            },
+            failOnStatusCode: false,
+          }).then($response =>{
+            expect($response.status).to.be.eq(201)
+            expect($response.body).to.have.property('id')
+            cy.deleteComment($response.body.id)
+          });
+    });
+    it('04_002: Should update a comment',()=> {

```

Comment:
When chaining two test cases that could be using the comment, you could store the ID of the newly created comment and edit it here, or later delete it.